inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/source/sortsdk/SortSdkSource.java

```diff
@@ -207,7 +220,7 @@ private void updateAllClientConfig() {
      * @param config The config to be updated.
      */
     private void updateClientConfig(SortClientConfig config) {
-        config.setManagerApiUrl(CommonPropertiesHolder.getSourceConfigManagerUrl());
+        config.setManagerApiUrl(ManagerAddrGetHandler.getSortSourceConfigUrl());
```

Comment:
When common.properties have the key "sortSourceConfig.QueryConsumeConfigType", the method "updateClientConfig" is unusable.

Reply:
in line 243, updateClientConfig is used before init sort client